### PR TITLE
Feature[TEO-336] Volunteer API integration

### PR DESCRIPTION
**Pull request by birloiflorian**
[TEO-336](https://wearetribus.atlassian.net/browse/TEO-336) Volunteer API Integration
[TEO-373](https://wearetribus.atlassian.net/browse/TEO-373) Location not mapped on Profile Card for Access Request

### CODE REVIEW

File: frontend/src/pages/AccessRequests.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const PendingAccessRequestsTableHeader = [
     selector: (row: IAccessRequest) =>
       row.requestedBy.location
         ? `${row.requestedBy.location?.name}, ${row.requestedBy.location?.county?.abbreviation}`
```
Comment (radulescuandrew) [inline frontend/src/pages/AccessRequests.tsx:74]: formatLocation

File: frontend/src/services/volunteer/volunteer.api.ts [hunk 2]
```diff
@@ -48,17 +48,11 @@ export const updateVolunteer = async (
...
-    branch: data.branch.value,
-    role: data.role.value,
-    department: data.department.value,
-  });
   return API.patch(`volunteer/${id}`, {
     ...data,
-    branch: data.branch.value,
-    role: data.role.value,
-    department: data.department.value,
+    branchId: data.branch?.key,
```
Comment (radulescuandrew) [inline frontend/src/services/volunteer/volunteer.api.ts:53]: Technical task: patch for update (send only changes)

**Review by radulescuandrew — CHANGES_REQUESTED** [review]
Test

File: frontend/src/services/volunteer/volunteer.api.ts (hunk 2 above)
Comment (birloiflorian) [reply frontend/src/services/volunteer/volunteer.api.ts:53]: Done

**Pull request merged**